isce2_topsapp/__main__.py

```diff
@@ -129,5 +133,84 @@ def main():
             aws.upload_file_to_s3(file, args.bucket, args.bucket_prefix)
 
 
+def gunw_burst():
+    parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--username')
+    parser.add_argument('--password')
+    parser.add_argument('--bucket')
+    parser.add_argument('--bucket-prefix', default='')
+    parser.add_argument('--dry-run', action='store_true')
+    parser.add_argument('--reference-scene', type=str, required=True)
+    parser.add_argument('--secondary-scene', type=str, required=True)
+    parser.add_argument('--image-number', type=int, required=True)
+    parser.add_argument('--burst-number', type=int, required=True)
+    parser.add_argument('--azimuth-looks', type=int, default=2)
+    parser.add_argument('--range-looks', type=int, default=10)
+    args = parser.parse_args()
+
+    ensure_earthdata_credentials(args.username, args.password)
+
+    ref_resp, sec_resp = get_asf_slc_objects([args.reference_scene, args.secondary_scene])
```

Comment:
```suggestion
    ref_obj, sec_obj = get_asf_slc_objects([args.reference_scene, args.secondary_scene])
```
I think `resp` here is response? Generally, I'd like the variables to match the fn signature and it's typically worth spelling things all the way out.